.github/workflows/build_publish_image_release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           if [[ "$TAG_NAME" =~ ^([^@]+)@([0-9]+\.[0-9]+\.[0-9]+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?)$ ]]; then
             # Sets and passes the imageName output from this step, which is used in the subsequent step
             # e.g. `steps.extract_tag.outputs.imageName` gets set to "wikibase" matched from "wikibase@1.0.1")
-            echo "::set-output name=imageName::${BASH_REMATCH[1]}"
+            echo "{imageName}=${BASH_REMATCH[1]}" >> $GITHUB_OUTPUT
```

Comment:
❤️

Reply:
Yeah still not the clearest API, but better.